python/paddle/fluid/layers/nn.py

```diff
@@ -5354,3 +5355,70 @@ def rank_loss(label, left, right, name=None):
                 "Right": right},
         outputs={'Out': out})
     return out
+
+
+def flatten(x, axis=1, name=None):
```

Comment:
what is this api design based on? Seems not the same as numpy.
https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.ndarray.flatten.html

Reply:
based on onnx operator Flatten